docs/source/examples/Widget Low Level.ipynb

```diff
@@ -516,11 +516,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "A template project is available in the form of a cookie cutter: https://github.com/jupyter-widgets/widget-cookiecutter\n",
+    "Two template projects is available in the form of cookiecutters\n",
     "\n",
-    "This project is meant to help custom widget authors get started with the packaging and the distribution of Jupyter interactive widgets.\n",
+    "- CommonJS: https://github.com/jupyter-widgets/widget-cookiecutter\n",
+    "- TypeScript: https://github.com/jupyter-widgets/widget-ts-cookiecutter\n",
     "\n",
-    "It produces a project for a Jupyter interactive widget library following the current best practices for using interactive widgets. An implementation for a placeholder \"Hello World\" widget is provided."
+    "These cookiecutters are meant to help custom widget authors get started with the packaging and the distribution of Jupyter interactive widgets.\n",
+    "\n",
+    "They produce a project for a Jupyter interactive widget library following the current best practices for using interactive widgets. An implementation for a placeholder \"Hello World\" widget is provided."
```

Comment:
Maybe "They produce a project for a Jupyter interactive widget library" -> "They produce a project for a widget library"

i.e. drop "Jupyter interactive". By now, people should know what we're talking about and the sentence feels a little stilted.